jobQueue/jobWrapper.js

```diff
@@ -130,7 +130,7 @@ function logJobOnComplete(req, job) {
       .then((resObj) => {
         logObject.user = resObj.username;
         logObject.token = resObj.tokenname;
-      });
+      }).catch(() => {}); // no-op
```

Comment:
I was thinking about this too but in other places where we are calling this, we do not do anything. So, I left it this way to be consistent.  Also, this will not happen in prod or any environment where a token is required. Let me know if you still want to log something here.